wordpress_importer/admin.py

```diff
@@ -5,3 +5,5 @@
 # Register your models here.
 admin.site.register(models.ImageImport)
 admin.site.register(models.PostImport)
```

Comment:
These are unrelated changes that have been pulled in, but aren't associated with another issue.